server/render/index.js

```diff
@@ -127,3 +127,17 @@ export function serverRender( req, res ) {
 		res.render( 'index.jade', context );
 	}
 }
+
+export function serverRenderError( err, req, res, next ) {
+	if ( err ) {
+		if ( config( 'env' ) !== 'production' ) {
+			console.error( err );
+		}
+		req.error = err;
+		res.status( err.status || 500 );
+		res.render( '500.jade', req.context );
+		return;
+	}
+
+	next();
```

Comment:
This is a terminating middleware by definition (since it `res.render()`s), so we shouldn't invoke `next()` here.

Reply:
I think error handling middleware are stackable so it shouldn't be an issue to keep this call.